doc/lxc.md

```diff
@@ -4,7 +4,7 @@
 (refer to https://wiki.debian.org/LXC)
 
 ```sh
-aptitude install lxc bridge-utils libvirt-bin debootstrap dnsmasq
+aptitude install lxc bridge-utils ebtables libvirt-bin debootstrap dnsmasq gnuplot
 ```
```

Comment:
OK. You can drop gnuplot. I added ebtables because it doesn't get pulled if you have APT::Install-Recommends disabled. I encountered ebtables missing errors.